cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -8,56 +8,57 @@
 
-   value.image:               (Optional) An object containing:
+   value.image:                  (Optional) An object containing:
 
-   value.image.is_decorative: Whether the image is a bg image or not.
-                              Default is false.
+   value.image.is_decorative:    Whether the image is a bg image or not.
+                                 Default is false.
 
-   value.image.is_square:     Whether the image is square or not.
-                              Default is false.
+   value.image.is_square:        Whether the image is square or not.
+                                 Default is false.
 
-   value.image.url:           String for the image URL.
+   value.image.url:              String for the image URL.
 
-   value.image.alt:           String for image alt text.
+   value.image.alt:              String for image alt text.
 
-   value.heading:             A string containing title of module.
+   value.heading:                A string containing title of module.
 
-   value.sub_heading:         A string containing the subtitle of module.
+   value.sub_heading:            A string containing the subtitle of module.
 
-   value.body:                A string containing the content to display below
-                              the heading (typically a paragraph).
+   value.body:                   A string containing the content to display
+                                 below the heading (typically a paragraph).
 
-   value.links:               A tuple of items to create a list of links,
-                              containing:
+   value.links:                  A tuple of items to create a list of links,
+                                 containing:
 
-   value.links[i].url:        A string for the URL of the link.
+   value.links[i].url:           A string for the URL of the link.
 
-   value.links[i].text:       A string for the text of the link.
+   value.links[i].text:          A string for the text of the link.
 
-   value.is_button:           Boolean for whether the link should be a button.
-                              Default is false.
+   value.is_button:              Boolean for whether the link should be
+                                 rendered as a button. Default is false.
 
-    value.should_link_image:  Boolean for whether the image should be wrapped in
-                              a tag.
+   value.link_image_and_heading: Boolean for whether the image and heading
+                                 should be linked to the value.links[0].url.
 
    ========================================================================== #}
 
 {% macro info_unit(value) %}
 {# TODO: If value.modifier is just to switch between two layouts,
          change to a boolean parameter. #}
+{% set link_image_and_heading = value.link_image_and_heading
+                                and value.links
+                                and value.links[0]
+                                and value.links[0].url %}
+
```

Comment:
This is interesting.  Ideally this logic would exist in a python function on the molecule's class, but I know this is how it was before so it doesn't need to be a blocker.